components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/SQLQueries.java

```diff
@@ -41,10 +41,15 @@ public class SQLQueries {
     public static final String CHECK_EXISTING_ATTRIBUTE_WITH_AUDIENCE_SQL =
             "SELECT TENANT_ID, ROLE_NAME, ATTR_NAME FROM IDN_SCIM_GROUP WHERE IDN_SCIM_GROUP.TENANT_ID=? AND " +
                     "IDN_SCIM_GROUP.ROLE_NAME=? AND IDN_SCIM_GROUP.ATTR_NAME=? AND IDN_SCIM_GROUP.AUDIENCE_REF_ID=?";
+
+    @Deprecated
```

Comment:
Can't we update this constant. As there is no column called `UM_ATTR_VALUE` which should be anyway wrong.